recipe/meta.yaml

```diff
@@ -27,7 +26,7 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.6
+    - python
     - mako
     - sqlalchemy >=1.3.0
     # Keeping noarch: python for now but may need to constrain these 2 as upstream
```

Comment:
Since we remove noarch, this comment is no longer needed.

Reply:
updated the pinnings for `Setuptools`:
https://github.com/AnacondaRecipes/alembic-feedstock/blob/a31bfe5ed4acbbb9916a0ed667603597bae149ad/recipe/meta.yaml#L26

Reply:
The unnecesary comments were removed:
https://github.com/AnacondaRecipes/alembic-feedstock/blob/a31bfe5ed4acbbb9916a0ed667603597bae149ad/recipe/meta.yaml#L31&L32